Makefile

```
@@ -9,7 +9,13 @@ precommit: test.dev
 install:
 	go mod tidy
 	npm install
+ifeq ($(OS), Windows_NT)
+	cd livebud
+	npm install
+	cd ..
+else
 	(cd livebud && npm install)
+endif
```

```suggestion
	@ # Note: We don't use a subshell here to support Windows. See #79.
	cd livebud
	npm install
	cd ..
```

I'd like to avoid conditionally checking your OS if we can. Since the windows version also works on linux and mac, let's just use that. I also just added a note to my future self not to change it back 😅

---

Definitely understandable!! I just pushed up that change 😋

---

Thanks @syke99! By the way, next time you can just hit the "Commit suggestion" button if the change works for you 😌

---

Ahh, didn't even see that option until now 😂 Thanks, I'll use it next time 😋